Firmware/ultralcd.cpp

```diff
@@ -3172,18 +3172,14 @@ void lcd_show_choices_prompt_P(uint8_t selected, const char *first_choice, const
     lcd_set_cursor(0, 3);
```

Comment:
Don't we need to update line 3168 `@param second_col column on LCD where second choice is rendered. If third choice is set, this value is hardcoded to 7` ?

Reply:
yes, third choice is hardcoded to 18 in this PR because it became a one-character button

Reply:
I’ll fix this in the evening 👍

Reply:
Fixed :) We no longer hardcode this value.